<launch>
  <arg name="robot_name"      default="dasom"/>
  <arg name="X_P_gain"          default="50"/>
  <arg name="X_I_gain"          default="50"/>
  <arg name="X_D_gain"          default="50"/>

  <arg name="Y_P_gain"          default="50"/>
  <arg name="Y_I_gain"          default="50"/>
  <arg name="Y_D_gain"          default="50"/>

  <arg name="VX_P_gain"         default="50"/>
  <arg name="VX_I_gain"         default="3"/>
  <arg name="VX_D_gain"         default="3"/>

  <arg name="VY_P_gain"         default="50"/>
  <arg name="VY_I_gain"         default="3"/>
  <arg name="VY_D_gain"         default="3"/>

  <arg name="torque_const_1"         default="3"/>
  <arg name="torque_const_2"         default="3"/>

  <arg name="offset_1"         default="3"/>
  <arg name="offset_2"         default="3"/>



  <param name="robot_name"    value="$(arg robot_name)"/>
  <param name="X_P_gain"        value="$(arg X_P_gain)"/>
  <param name="X_I_gain"        value="$(arg X_I_gain)"/>
  <param name="X_D_gain"        value="$(arg X_D_gain)"/>

  <param name="Y_P_gain"        value="$(arg Y_P_gain)"/>
  <param name="Y_I_gain"        value="$(arg Y_I_gain)"/>
  <param name="Y_D_gain"        value="$(arg Y_D_gain)"/>

  <param name="VX_P_gain"        value="$(arg VX_P_gain)"/>
  <param name="VX_I_gain"        value="$(arg VX_I_gain)"/>
  <param name="VX_D_gain"        value="$(arg VX_D_gain)"/>

  <param name="VY_P_gain"        value="$(arg VY_P_gain)"/>
  <param name="VY_I_gain"        value="$(arg VY_I_gain)"/>
  <param name="VY_D_gain"        value="$(arg VY_D_gain)"/>

  <param name="torque_const_1"        value="$(arg torque_const_1)"/>
  <param name="torque_const_2"        value="$(arg torque_const_2)"/>

  <param name="offset_1"        value="$(arg offset_1)"/>
  <param name="offset_2"        value="$(arg offset_2)"/>

  <node name="torque_jacobian_velocity.cpp" pkg="two_link" type="torque_jacobian_velocity.cpp" required="true" output="screen"/>

</launch>
